exchanges/bybit/bybit_types.go

```diff
@@ -2035,3 +2036,26 @@ type Error struct {
 	ExtCode         string `json:"ext_code"`
 	ExtMsg          string `json:"ext_info"`
 }
+
+// AccountTypeHolder holds the account type associated with the loaded API key.
+type AccountTypeHolder struct {
```

Comment:
Any need for the holder to be exported given no pointer funcs and no accessible properties?